tests/std/tests/P2502R2_generator_promise/test.cpp

```diff
@@ -186,37 +191,49 @@ void test_one() {
     }
 }
 
-template <class Ref, class V = void>
+template <class Ref, class V, bool TestingIncomplete = false>
 void test_with_allocator() {
-    test_one<Ref, V>();
-    test_one<Ref, V, allocator<void>>();
-    test_one<Ref, V, pmr::polymorphic_allocator<void>>();
-    test_one<Ref, V, StatelessAlloc<void>>();
-    test_one<Ref, V, StatelessAlloc<void, false_type>>();
-    test_one<Ref, V, StatelessAlloc<void, true_type, int>>();
-    test_one<Ref, V, StatelessAlloc<void, false_type, int>>();
+    test_one<Ref, V, void, TestingIncomplete>();
+    test_one<Ref, V, allocator<void>, TestingIncomplete>();
+    test_one<Ref, V, pmr::polymorphic_allocator<void>, TestingIncomplete>();
+    test_one<Ref, V, StatelessAlloc<void>, TestingIncomplete>();
+    test_one<Ref, V, StatelessAlloc<void, false_type>, TestingIncomplete>();
+    test_one<Ref, V, StatelessAlloc<void, true_type, int>, TestingIncomplete>();
+    test_one<Ref, V, StatelessAlloc<void, false_type, int>, TestingIncomplete>();
 }
 
-template <class T>
+template <class T, bool TestingIncomplete = false>
 void test_with_type() {
-    test_with_allocator<T>();
-    test_with_allocator<T&>();
-    test_with_allocator<const T&>();
-    test_with_allocator<T&&>();
-    test_with_allocator<const T&&>();
-
-    test_with_allocator<Proxy<T>, T>();
-    test_with_allocator<Proxy<T>&, T>();
-    test_with_allocator<const Proxy<T>&, T>();
-    test_with_allocator<Proxy<T>&&, T>();
-    test_with_allocator<const Proxy<T>&&, T>();
+    test_with_allocator<T, void, TestingIncomplete>();
+    test_with_allocator<T&, void, TestingIncomplete>();
+    test_with_allocator<const T&, void, TestingIncomplete>();
+    test_with_allocator<T&&, void, TestingIncomplete>();
+    test_with_allocator<const T&&, void, TestingIncomplete>();
+
+    test_with_allocator<Proxy<T>, T, TestingIncomplete>();
+    test_with_allocator<Proxy<T>&, T, TestingIncomplete>();
+    test_with_allocator<const Proxy<T>&, T, TestingIncomplete>();
+    test_with_allocator<Proxy<T>&&, T, TestingIncomplete>();
+    test_with_allocator<const Proxy<T>&&, T, TestingIncomplete>();
 }
 
+#ifndef _M_CEE // TRANSITION, VSO-1659496

```

Comment:
No change requested: Technically, this isn't necessary because `<generator>` is C++23, and `/clr` is currently limited to C++20. However, following the usual pattern seems fine.